plural/helm/kas/templates/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "kas.configMapName" . }}
+  labels:
+    {{- include "kas.labels" . | nindent 4 }}
+data:
+  config.yaml: |
+    agent:
+      listen:
+        address: ":{{ .Values.service.internalPort }}"
+        websocket: true
+        certificate_file: "/etc/kas/tls.crt"
+        key_file: "/etc/kas/tls.key"
+    kubernetes_api:
+      listen:
+        address: ":{{ .Values.service.kubernetesApiPort }}"
+        certificate_file: "/etc/kas/tls.crt"
+        key_file: "/etc/kas/tls.key"
+      url_path_prefix: "{{ .Values.ingress.kubernetesApiPath }}"
+    observability:
+      listen:
+        address: ":{{ .Values.service.observabilityPort }}"
+    redis:
+      server:
+        address: "{{ printf "%s-master" (include "common.names.fullname" (index .Subcharts "redis")) }}:{{ .Values.redis.master.service.ports.redis }}"
+      key_prefix: {{ include "kas.name" . }}
+      password_file: "/etc/kas/redis_server_secret"
+    api:
+      listen:
+        address: ":{{ .Values.service.apiInternalPort }}"
+        authentication_secret_file: "/etc/kas/.api_secret"
+        certificate_file: /etc/kas/tls.crt
```

Comment:
i'm not totally sure but if the ingress is configured to terminate tls are these necessary?

Reply:
Yep, it was just a leftover. Cleaned up now.